src/chrome/content/rules/Jimdo-Server.com.xml

```diff
@@ -0,0 +1,28 @@
+
+<!--
+	Other Jimdo rulesets:
+		-Jimdo.xml
+		-Jimcdn.com.xml
+		-Jimstatic.com.xml
+		-Jimdo.design.xml
+
+	Nonfunctional subdomains:
+		jimdo-server.com
```

Comment:
This is not very explicit. Please instead say what kind of error you get. In this case `Invalid certificate`